src/cascadia/ut_app/JsonTests.cpp

```diff
@@ -107,7 +107,7 @@ namespace TerminalAppUnitTests
         for (size_t i = 0; i < expectedCampbellTable.size(); i++)
         {
             const auto& expected = expectedCampbellTable.at(i);
-            const auto& actual = scheme.GetTable().at(i);
+            const auto& actual = til::color{ scheme->Table().at(static_cast<uint32_t>(i)) };

```

Comment:
this is invalid -- you've captured a reference to a local temporary.

you want this:

```suggestion
            const til::color actual{ scheme->Table().at(static_cast<uint32_t>(i)) };
```